### ISSUE TITLE
[fifo] Handle 404 when fetching last message.

### ISSUE THREAD

**Issue by synrg** [bobloy/Fox-V3#166]
**Describe the bug**

A task I defined yesterday that was scheduled to execute this morning didn't execute.

A 404 was raised by discord/http.py. A `try/except` block should enclose the `fetch_message(channel.last_message_id)` in `task.py` at line 255 instead of just checking if it returns None.

**To Reproduce**

```
,fifo add hybrids ebird hybrids
,fifo addtrigger cron hybrids 0 5 * * *
,fifo set hybrids #hybrids-alerts-ma
,fifo printschedule
```

`fifo printschedule` outputs:

```
Jobstore default:

_execute_task (trigger: cron[month='', day='', day_of_week='*', hour='5', minute='0'], next run at: 2021-01-04 05:00:00 AST)
```

**Expected behavior**

The task should have executed.

**Screenshots or Error Messages**

![image](https://user-images.githubusercontent.com/1204376/103523288-04d0a700-4e52-11eb-973e-73c50d8c1478.png)

**Additional context**

The channel #hybrids-alerts-ma has messages in it, but frequently the *only* new messages added to it are from the scheduled task. Previously, we used Sinbad's scheduler cog. Yesterday we switched to fifo. In doing so, I very likely typed, then deleted, a command or two in the #hybrids-alerts-ma channel before remembering that `fifo` allows one to type the commands in another channel, and use `,fifo set` afterwards to set the channel for the task to execute in. No new messages were added to the channel since yesterday's switch to the new cog, setting up the conditions for failure this morning.

**Comment by synrg** [comment]
The error raised from `discord/http.py:243` was:

```
NotFound: 404 Not Found (error code: 10008): Unknown Message
```

**Comment by bobloy** [comment]
Thank you for reporting this. The next update to FIFO will include a fix.

**Comment by bobloy** [comment]
Should be corrected in #167
